overlord/configstate/configcore/refresh.go

```diff
@@ -40,6 +40,19 @@ func init() {
 	supportedConfigurations["core.refresh.rate-limit"] = true
 }
 
+func reportOrResetInvalidManageRefreshes(tr config.Conf, optName string) error {
+	// check if the option is set as part of transaction changes; if not than
+	// it's already set in the config state.
+	for _, k := range tr.Changes() {
+		if k == "core."+optName {
+			return fmt.Errorf("cannot set schedule to managed")
+		}
+	}
```

Comment:
given that afaik we don't have at any other option that can be invalidated after being set? (is this true?) doing something only here seems ok

Reply:
we probably want to expand the comment a bit though but how depends a bit on what we decided to do about my other points

Reply:
Done.